_components/accordions.md

```diff
@@ -112,8 +132,12 @@ title: Accordions
 
 </div>
 
-<div class="usa-grid">
-  <div class="usa-width-one-half">
+<div class="usa-accordion-bordered">
+  <button class="usa-button-unstyled usa-accordion-button"
+      aria-expanded="true" aria-controls="collapsible-0">
+    Documentation
+  </button>
+  <div id="collapsible-0" aria-hidden="false" class="usa-accordion-content">
     <h3>Use</h3>
```

Comment:
@msecret we need to add in `usa-heading` class to all <h3> headings in the documentation in order to get the blue heading.